tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -514,6 +514,21 @@ exports[`show detailed usage with --help write (stdout) 1`] = `
 
 exports[`show detailed usage with --help write (write) 1`] = `Array []`;
 
+exports[`show detailed usage with plugin options (stderr) 1`] = `""`;
+
+exports[`show detailed usage with plugin options (stdout) 1`] = `
+"--tab-width <int>
+
+  Number of spaces per indentation level.
+
+Default: 2
+Plugin defaults:
+* ../plugins/automatic/node_modules/prettier-plugin-bar: 4
```

Comment:
Could we use the `package.json` name here? (Happy to do this in a follow up PR though)

Reply:
That's the result from `--plugin=../plugins/automatic/node_modules/prettier-plugin-bar`, I guess it should be fine?

- `--plugin=path/to/plugin.js` (path)
   
  ```
  Plugin defaults:
  * path/to/plugin.js: 4
  ```

- `--plugin=prettier-plugin-bar` (package name)
   
  ```
  Plugin defaults:
  * prettier-plugin-bar: 4
  ```

Reply:
Ah, cool. I'm happy with that. Does it work for plugins that are automatically resolved?

Reply:
It should work. @czosel could you add a test for that? just clone the test case and set its cwd to `path/to/plugins/automatic` then remove the `--plugin=xxx` part.

Reply:
done!